File: kotlinames-common/src/main/java/com/nagopy/android/kotlinames/property/nullable/KNullableShortPropertyName.java
package com.nagopy.android.kotlinames.property.nullable;

import com.nagopy.android.kotlinames.KNullablePropertyName;
import com.nagopy.android.kotlinames.KSortablePropertyName;

public class KNullableShortPropertyName implements KNullablePropertyName<Short>, KSortablePropertyName<Short> {

    private final String name;

    public KNullableShortPropertyName(String name) {
        this.name = name;
    }

    @Override
    public String name() {
        return name;
    }
}

File: kotlinames/src/main/kotlin/com/nagopy/android/kotlinames/property/nullable/KNullableShortPropertyName.kt
package com.nagopy.android.kotlinames.property.nullable

import com.nagopy.android.kotlinames.KNullablePropertyName
import com.nagopy.android.kotlinames.KSortablePropertyName

class KNullableShortPropertyName(val name: String) : KNullablePropertyName<Short>, KSortablePropertyName<Short> {

    override fun name(): String = name

}
